src/NzbDrone.Core/Indexers/Rarbg/RarbgParser.cs

```diff
@@ -28,6 +33,11 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
             var jsonResponse = new HttpResponse<RarbgResponse>(indexerResponse.HttpResponse);
 
+            if (jsonResponse.Resource.rate_limit == 1)
```

Comment:
remove this for now.  RarBG has a bug where rate_limit = 1 on almost every request

Ref - https://github.com/Prowlarr/Prowlarr/issues/1277
Ref - https://github.com/Prowlarr/Prowlarr/commit/5cc044aa8fbc7ba62d9c32442d8125b9b347d28a